### PR TITLE
Fix pagination alignment

### PR DESCRIPTION
**Pull request by Ncode-93**
## UPDATE: Fix Pagination Alignment
This PR addresses the issue of pagination layout misalignment between 1-digit, 2-digit, and 3-digit page numbers, which was causing inconsistent spacing in the pagination component. The changes ensure that page numbers remain consistently aligned regardless of the number of digits displayed (e.g., page 1, page 10, page 100).

## Current behavior (updates):
--> Adjusted the pagination component's styling to ensure that page numbers of varying digit lengths (1-digit, 2-digit, 3-digit) are consistently aligned.
--> Introduced flexible padding and margin adjustments to handle cases where digit lengths change as the user navigates through different pages.
--> Reviewed and tested on various screen sizes to ensure the layout remains responsive.

## New behavior
Consistent pagination with no breaks for 1,2 or 3 digit numbers.

## Is this a breaking change (Yes/No): NO

### CODE REVIEW

**Comment by Ncode-93** [comment]
@illionillion Have a look

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@ node-jiti
 
 # dependencies
 node_modules
+package-lock.json
```
Comment (illionillion) [inline .gitignore:7]: This is not necessary.
lock file is not included in the diff if the library is not added.

File: contents/navigation/paginations/right-justified-with-page-position/index.tsx
Comment (illionillion) [inline contents/navigation/paginations/right-justified-with-page-position/index.tsx:1]: Do not use the `span` tag directly.

```tsx
import { Text, Pagination, Flex, Box, ui } from "@yamada-ui/react"
import { useState, type FC } from "react"

const RightJustifiedWithPagePosition: FC = () => {
  const [page, onChange] = useState<number>(1)

  return (
    <Box w="full">
      <Flex justifyContent={{ base: "end", sm: "center" }}>
        <Pagination
          page={page}
          onChange={onChange}
          total={10}
          w="md"
          size={{ base: "md", sm: "sm", xs: "xs" }}
          variant="ghost"
        />
      </Flex>
      <Text
        fontSize="sm"
        color={["gray.700", "gray.200"]}
        textAlign="right"
        m="md"
      >
        Showing{" "}
        <ui.span
          display="inline-block"
          minWidth="20px"
          textAlign="center"
        >
          {page * 10 - 9}
        </ui.span>
        -
        <ui.span
          display="inline-block"
          minWidth="30px"
          textAlign="center"
        >
          {page * 10}
        </ui.span>{" "}
        of 100
      </Text>
    </Box>
  )
}

export default RightJustifiedWithPagePosition
```

File: contents/navigation/paginations/simple-button/index.tsx
Comment (illionillion) [inline contents/navigation/paginations/simple-button/index.tsx:1]: Again, do not use `span` tags directly.

**Review by illionillion — CHANGES_REQUESTED** [review]
@Ncode-93 
Thank for PR!!

Correction please.

File: contents/navigation/paginations/simple-button/index.tsx
Comment (illionillion) [inline contents/navigation/paginations/simple-button/index.tsx:1]: add `justifyContent="center"` in `Pagenation`

Comment (Ncode-93) [reply contents/navigation/paginations/simple-button/index.tsx:1]:
```
<Pagination
        page={page}
        onChange={onChange}
        total={10}
        justifyContent={center}
        innerProps={{ display: "none" }}
        controlPrevProps={{
          children: (
            <Button variant="ghost" size="sm">
              Prev
            </Button>
          ),
        }}
        controlNextProps={{
          children: (
            <Button variant="ghost" size="sm">
              Next
            </Button>
          ),
        }}
      />
```
Thanks for pointing out. I think this is the pagination you are talking about.

Comment (illionillion) [reply contents/navigation/paginations/simple-button/index.tsx:1]:
```tsx
<Pagination
        page={page}
        onChange={onChange}
        total={10}
        justifyContent="center"
        innerProps={{ display: "none" }}
        controlPrevProps={{
          children: (
            <Button variant="ghost" size="sm">
              Prev
            </Button>
          ),
        }}
        controlNextProps={{
          children: (
            <Button variant="ghost" size="sm">
              Next
            </Button>
          ),
        }}
      />
```

**Comment by Ncode-93** [comment]
Hey @illionillion I have made the required changes. Have a look.

**Review by illionillion — APPROVED** [review]
LGTM!

**Review by hirotomoyamada — APPROVED** [review]

**Comment by illionillion** [comment]
Closes #506

**Pull request merged**